app/src/main/java/co/nimblehq/compose/ecommerce/ui/category/Categories.kt

```diff
@@ -0,0 +1,55 @@
+package co.nimblehq.compose.ecommerce.ui.category
```

Comment:
What do you think if we rename the class name to have a suffix with `Compose` e.g. `CategoriesCompose` like  `CategoriesActivity` or `CategoriesFragment` 🙏

Reply:
Agree, having Compose postfix would help us to differentiate them to model classes also. We should consider to have it in team convention.